src/view/configuration/configuration.jsx

```diff
@@ -138,32 +134,10 @@ const validationSchema = object().shape({
   )
 });
 
-const Configuration = ({
-  initInfo,
-  formikProps,
-  registerImperativeFormApi
-}) => {
-  const { values } = formikProps;
-  const reportAsyncError = useReportAsyncError();
+const Configuration = ({ initInfo }) => {
+  const [{ value: instances }] = useField("instances");
   const [selectedTabKey, setSelectedTabKey] = useState(0);
 
-  useEffect(async () => {
-    registerImperativeFormApi({
-      getSettings,
-      formikStateValidationSchema: validationSchema
-    });
-    let initialValues;
-    try {
-      initialValues = await getInitialValues({
-        initInfo
-      });
-    } catch (e) {
-      reportAsyncError(e);
-      return;
-    }
```

Comment:
I think we need to take into consideration error handling for the `getInitialValues` call that we're now doing in ExtensionView. If anything goes wrong in `getInitialValues`, we should report an async error, I believe.